db/seeds.rb

```diff
@@ -24,5 +24,13 @@
   else
     flag = false
   end
-  patient.pregnancies.create({last_menstrual_period_time: DateTime.new(2016,1,1), urgent_flag: flag})
+  pregnancy = patient.pregnancies.create({last_menstrual_period_time: DateTime.new(2016,1,1), urgent_flag: flag})
+  8.times do
+    pregnancy.calls.create({status: 'Left voicemail', creating_user_id: user.id})
+  end
+  if(patient.name[-1, 1].to_i == 0) then
```

Comment:
Not terribly clear what this is doing -- looks like it's adding five calls to Patient 0, but I had to read it a few times to figure it out. 

Can we make it look a little more 'ruby'? We could either rewrite the `if` statement to be more clear about when it's executing the block (something like `if patient.name == 'Patient 0'`) or pop it out of the block entirely and add a line to the effect of `10.times do Patient.find(name: 'Patient 0').pregnancy.calls.create ...`. 

Sorry to be nitpicky about this!